hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMDatanodeProtocolServer.java

```diff
@@ -280,6 +280,29 @@ public static SCMRegisteredResponseProto getRegisteredResponse(
     return cmd.getProtoBufMessage();
   }
 
+  private String constructCommandAuditMap(List<SCMCommandProto> cmds) {
+    StringBuilder auditMap = new StringBuilder();
+    auditMap.append("[");
+    for (SCMCommandProto cmd : cmds) {
+      if (cmd.getCommandType().equals(deleteBlocksCommand)) {
+        auditMap.append("commandType: ").append(cmd.getCommandType());
+        auditMap.append(" deleteTransactionsCount:");
+        auditMap.append(cmd.getDeleteBlocksCommandProto().getDeletedBlocksTransactionsList().size());
+      } else {
+        auditMap.append(cmd);
+      }
+      auditMap.append(" , ");
+    }
+    int len = auditMap.toString().length();
```

Comment:
Let's not call `toString()` twice.

```suggestion
    int len = auditMap.length();
```